source/options.tsx

```diff
@@ -173,10 +174,37 @@ async function highlightNewFeatures(): Promise<void> {
 	void browser.storage.local.set({featuresAlreadySeen});
 }
 
+async function getFeaturesDisabledViaHotfix(): Promise<HTMLElement[]> {
+	const hotfixes = await cache.get<string[][]>('hotfixes');
+	if (!hotfixes) {
+		return [];
+	}
+
+	const {version} = browser.runtime.getManifest();
+	const featureNames: Set<string> = new Set(features.map(({id}) => id));
+	const disabledFeatures = [];
+	for (const [feature, unaffectedVersion] of hotfixes) {
+		const relatedIssue = '#1234';
+		if (featureNames.has(feature) && (version === '0.0.0' || !unaffectedVersion || compareVersions(unaffectedVersion, version) > 0)) {
```

Comment:
`version === '0.0.0'` is constant, it mean it needs to be checked once at the top of this function

Reply:
What I meant is that it should be an early return. If this, then return []

Reply:
Oh so the notice shouldn't be shown at all while in dev mode? I think it could be useful though, especially with the links to the issues

Reply:
Oh I'm reading the condition now. You're _showing_ the notice, not hiding it. In this case it would be wrong because `0.0.0` does not disable any feature, so it would say something incorrect

Reply:
Fixed in [`22e99ee` (#4394)](https://github.com/sindresorhus/refined-github/pull/4394/commits/22e99ee9a7302b182a3a16c967a6dcc51dbb288f)